### PR TITLE
Notify Dependent Applications When SimplyCop Is Updated

### PR DESCRIPTION
**Pull request by apeacock1991**
To make sure the style guide is kept in sync across repositories, I've configured dependabot notifications to all repos that currently implement the new SimplyCop.

FYI @simplybusiness/limited-liability 

- [x] Engineering

### CODE REVIEW

File: _pipeline/step_notify_dependent_apps.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+source "/opt/sb/sb-pipeline.sh"
+set -e
+
+LIST_OF_APPS="chopin,model_viewer"
```
Comment (budmc29) [inline _pipeline/step_notify_dependent_apps.sh]: No OK?

Comment (apeacock1991) [reply _pipeline/step_notify_dependent_apps.sh]: It's not using SimplyCop yet 😅

Comment (sebastian-nanek) [reply _pipeline/step_notify_dependent_apps.sh]: Not yet, but it's in our backlog.

Comment (budmc29) [reply _pipeline/step_notify_dependent_apps.sh]: 👍

**Pull request merged**